src/compas/rpc/proxy.py

```diff
@@ -56,6 +56,12 @@ class Proxy(object):
     capture_output : bool, optional
         If True, capture the stdout/stderr output of the remote process.
         In general, `capture_output` should be True when using a `pythonw` as executable (default).
+    path : str, optional
+        Path to the folder containing the module to be proxied.
+        This is useful for cases where the module to be proxied is not on the PYTHONPATH.
+    cwd : str, optional
+        Current working directory for the process that will be started to run the server.
+        This is useful for cases where a custom service is used and the service is not on the PYTHONPATH.
```

Comment:
Isn't `cwd` too cryptic for most people? Shouldn't we just err on the side of verbosity and call it `working_directory`?